src/pages/home.html

```diff
@@ -0,0 +1,1926 @@
+                        src="/src/assets/images/home/img-order5.png"
+                        alt="عکس سفارشی"
+                        class="h-[220px] w-[221px]"
+                    />
+                    <img
+                        src="/src/assets/images/home/img-order4.png"
+                        alt="عکس سفارشی"
+                        class="h-[220px] w-[221px] scale-[1.2]"
+                    />
+                    <img
+                        src="/src/assets/images/home/img-order3.png"
+                        alt="عکس سفارشی"
+                        class="h-[220px] w-[221px]"
+                    />
+                    <img
+                        src="/src/assets/images/home/img-order2.png"
+                        alt="عکس سفارشی"
+                        class="h-[220px] w-[221px]"
+                    />
+                    <img
+                        src="/src/assets/images/home/img-order1.png"
+                        alt="عکس سفارشی"
+                        class="h-[220px] w-[221px]"
+                    />
+                    <img
+                        src="/src/assets/images/home/diamond-shape.png"
+                        alt="شکل لوزی"
+                        class="absolute -right-[3%] -top-[21%] -z-10"
+                    />
+                    <div
+                        class="absolute -right-[2%] h-[240px] w-[159px] bg-gradient-to-l from-white from-0% to-transparent to-[100%]"
+                    ></div>
+                    <div
+                        class="absolute -left-[2%] h-[240px] w-[159px] bg-gradient-to-r from-white from-0% to-transparent to-[100%]"
+                    ></div>
+                </div>
+            </section>
+            <!-- orders section -->
+
+            <!-- features section -->
+            <section class="mx-auto mb-20 max-w-7xl py-24">
+                <div class="relative flex items-center justify-center">
+                    <img
+                        src="/src/assets/images/home/blues.png"
+                        alt=""
+                        class="absolute -top-[15%] right-[38%] h-[15px] w-[25px]"
+                    />
+                    <h3 class="text-5xl font-bold">چرا عکسخونه؟</h3>
+                </div>
+                <div class="mt-8 flex items-center justify-center gap-2">
+                    <img
+                        src="/src/assets/images/light-shape.png"
+                        alt=""
+                        class="rotate-180"
+                    />
+                    <p class="text-lg tracking-wider text-bali-hai">
+                        خوشحالیم که در ذهن شما ماندگاریم
+                    </p>
+                    <img
+                        src="/src/assets/images/light-shape.png"
+                        alt=""
+                        class=""
+                    />
+                </div>
+                <div
+                    class="mt-7 grid grid-cols-4 items-center justify-center gap-[35px]"
```

Comment:
All children should have the same height. It may be a good idea to use `flex items-stretch` here.